mobile.css

```diff
@@ -277,6 +277,142 @@ button,
   justify-content: center;
 }
 
+#modal-container {
+  display: block;
+  position: fixed;
+  top: 0;
+  left: -100%;
+  background-color: #191765cc;
+  width: 100vw;
+  height: 100vh;
+  opacity: 80%;
+  transition: all 1s ease-out;
+}
+
+#modal-container.show {
+  display: block;
+  opacity: 100%;
+  left: 0;
+  pointer-events: auto;
+  overflow-y: auto;
+}
+
+#modal {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  margin: 2em 1.5em 1em;
+  width: auto;
+  height: fit-content;
+  background-color: rgba(255, 255, 255, 0.2);
+  mix-blend-mode: screen;
+  box-shadow: 0 48px 48px rgba(37, 47, 137, 0.08);
+  border: 1px solid white;
+}
```

Comment:
- [ ] Please consider to add blur effect in the back of your model to not make conflict with the content of your portfolio.

| Your design  | Figma design|
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/53402600/174160297-dfa3c3de-a331-475c-8d91-6bd46917376f.png)  | ![image](https://user-images.githubusercontent.com/53402600/174160375-45256c81-5cc7-4728-b0f6-175faaf40a24.png)|

Reply:
@omarsalem7 Hi! Thank you so much. Honestly, I don't know how to do this without blurring the content inside the container. Could you please explain this to me?  :pray:

Reply:
@omarsalem7 Done :heavy_check_mark: